libs/datamodel/core/tests/renderer/enums.rs

```diff
@@ -0,0 +1,21 @@
+use indoc::indoc;
+
+#[test]
+fn enum_rendering_works() {
```

Comment:
Could this test be more of making the `@@map` indentation more screwed first, then expecting that it's evenly positioned to the correct indentation?

Reply:
It's going through the dml so I don't think it's necessary for confidence, but I'll do that :+1: